### ISSUE TITLE
Autocomplete 4.0 RC3 Wrong ElementRef for Multiselect

### ISSUE THREAD

**Issue by andelz** [primefaces/primeng#2544]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
``
**Current behavior**
I'm getting an error, when using backspace on Autocomplete input. 

"TypeError: Cannot read property 'nativeElement' of undefined."

After having a look to the components source code, I think there is an error in line 264. In case of multiselect mode you are checking the value of the inputEL property which in this case is always undefined (should be multiInputEl instead, I guess)

**Minimal reproduction of the problem with instructions**
Juust try to remove an item from a multiselect autocomplete component using Backspace.

* **Angular version:** 4.0.X

* **PrimeNG version:** 4.0.0 rc3

* **Browser:** all
 
* **Language:** all
![unbenannt-1](https://cloud.githubusercontent.com/assets/27723314/25122478/61381d94-2425-11e7-9123-0eb44e7df2e5.jpg)

**Comment by codechamploo** [comment]
I can confirm the same issue, and the issue is exactly where stated.  Thanks for reporting this.

**Comment by Mrtcndkn** [comment]
Can you create a plunkr or share some code according to your issue?

**Comment by codechamploo** [comment]
Hopefully this will suffice:
https://plnkr.co/edit/B1WaXOweVYhce1XAC9eo

The error and fix is mentioned by andelz:

After having a look to the components source code, I think there is an error in line 264. In case of multiselect mode you are checking the value of the inputEL property which in this case is always undefined (should be **multiInputEl** instead, I guess)

**Comment by andelz** [comment]
Thanks for that plunkr codechamploo, that's exactly what I meant.

**Comment by Mrtcndkn** [comment]
It is now fixed for 4.0.0-Final.
